MapboxNavigation/RouteVoiceController.swift

```diff
@@ -61,6 +54,18 @@ public class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
     
     
     /**
+     If true, a noise indicating the user is going to be rerouted will play prior to rerouting.
+     */
+    public var playRerouteSound = false
+
+    
+    /**
+     Sound to play prior to reroute. Inherits volume level from `volume`.
+     */
+    public var rerouteSoundPlayer: AVAudioPlayer = try! AVAudioPlayer(data: NSDataAsset(name: "reroute-sound", bundle: Bundle.navigationUI)!.data, fileTypeHint: AVFileTypeMPEGLayer3)
```

Comment:
I'm fine with exposing an audio player here but curious what the reasoning behind not exposing sound data is? I assume it would be the fileTypeHint.